cmd/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/gojp/goreportcard/check"
+	"github.com/gojp/goreportcard/handlers"
+)
+
+var allScores []handlers.Score
+
+func main() {
+	dir := "."
+	filenames, skipped, err := check.GoFiles(dir)
+	if err != nil {
+		log.Fatalf("could not get filenames: %v", err)
+	}
+	if len(filenames) == 0 {
+		log.Fatalf("no .go files found")
+	}
+
+	err = check.RenameFiles(skipped)
+	if err != nil {
+		log.Println("Could not remove files:", err)
+	}
+	defer check.RevertFiles(skipped)
+
+	checks := []check.Check{
```

Comment:
how much of this do we want to share between the command and the handler? might not be worth sharing a ton of the code b/c it's so small and i like being able to see all the logic in one place. opinions welcome